asset/js/ets_woo_qa_script.js

```diff
@@ -23,7 +23,7 @@ jQuery(document).ready(function(){
         }); 
 	});
 
-	jQuery("#ques-text-ar").on("click", function(){ 
+	jQuery(".ets-qa-textarea").on("click", function(){ 
 		jQuery(".ets-display-message").text(""); 
```

Comment:
Similarly, this will clear all messages of different questions forms.

Reply:
yes